.storybook/config.js

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import { configure, setAddon, addDecorator } from '@storybook/react';
 import infoAddon, { setDefaults } from '@storybook/addon-info';
+import { setOptions } from '@storybook/addon-options';
 import { checkA11y } from 'storybook-addon-a11y';
 import Container from './Container';
+import { version } from '../package.json';
+
+setOptions({
+  name: `carbon components react v${version}`,
```

Comment:
Discussed within the team, and we think it’d make more sense to just say “Carbon components React” or a like (one without a version) - Thanks!